OsmAnd/src/net/osmand/plus/resources/ResourceManager.java

```diff
@@ -937,6 +939,87 @@ public boolean containsAmenityRepositoryToSearch(boolean searchByName) {
 		return false;
 	}
 
+
+	private List<Amenity> searchRouteByName(String multipleSearch, double lat, double lon, boolean force, ResultMatcher<Amenity> matcher) {
+		List<AmenityIndexRepositoryBinary> list = new ArrayList<>();
+		List<Amenity> result = new ArrayList<>();
+		for (AmenityIndexRepository index : getAmenityRepositories(false)) {
+			if (index instanceof AmenityIndexRepositoryBinary) {
+				if (index.getFile().getName().contains("World")) {
```

Comment:
why do we exclude world maps? If this is going to be used for multipolygon thats not correct, for routes not sure but exclude reason is not clear